src/RwaToken.sol

```diff
@@ -0,0 +1,50 @@
+pragma solidity 0.5.12;
+
+contract RwaToken {
+    // --- ERC20 Data ---
+    string  public constant name     = "RWA Example Token";
+    string  public constant symbol   = "RWA001";
+    uint8   public constant decimals = 18;
+    uint256 public totalSupply;
+
+    mapping (address => uint)                      public balanceOf;
+    mapping (address => mapping (address => uint)) public allowance;
+
+    event Approval(address indexed src, address indexed guy, uint wad);
+    event Transfer(address indexed src, address indexed dst, uint wad);
+
+    // --- Math ---
+    function add(uint x, uint y) internal pure returns (uint z) {
+        require((z = x + y) >= x);
+    }
+    function sub(uint x, uint y) internal pure returns (uint z) {
+        require((z = x - y) <= x);
+    }
+
+    constructor() public {
+        balanceOf[msg.sender] = 1 ether;
+    }
+
+    // --- Token ---
+    function transfer(address dst, uint wad) external returns (bool) {
+        return transferFrom(msg.sender, dst, wad);
+    }
+    function transferFrom(address src, address dst, uint wad)
+        public returns (bool)
```

Comment:
This should be `external` to save memory copy gas.

Reply:
See above.